recipes/misaka/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{%set name = "misaka" %}
+{%set version = "2.0.0" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "336ef1381ab840046b7da9f95c4c28af17e636aed8dcfcf1efe972db05f73604" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cffi >=1.0.0
```

Comment:
Needs `msinttypes` for Windows only. (That's the AppVeyor Python 2.7 [error]( https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.6840/job/gv0a5ii97027lebc#L374 ).)